sgt-back-end/index.js

```diff
@@ -0,0 +1,119 @@
+const pg = require('pg');
+const port = 3000;
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+const express = require('express');
+const app = express();
+const ok = require('./queries');
+app.use(express.json());
+
+app.get('/api/grades', (req, res) => {
+
+  db.query(ok.GET)
+    .then(results => {
+      const all = results.rows;
+      res.status(200).json(all);
+    })
+    .catch(error => {
+      console.error(error);
+      res.status(500).send(error);
+    });
+});
+app.get('/api/grades/:gradeId', (req, res) => {
+  const gradeId = Number(req.params.gradeId);
+  if (!Number.isInteger(gradeId) || gradeId <= 0) {
+    res.status(400).json({ error: 'grade id must be a positive integer' });
+  }
+  db.query(ok.doesGradeIdExist, [gradeId])
+    .then(result => {
```

Comment:
Please put `()` around the arrow function parameter. Oh, I made this comment on one of your prior PRs, didn't I? You weren't here when we discussed this. The code is much easier to read if you always use `()`. You need to update your lint rules, though. One of the other students can help you with this.